README.md

```diff
@@ -328,6 +334,13 @@ The default values for each rule are described below.
 
 ##### shared
 
+
+###### operations
+| Rule                         | Default |
+| ---------------------------- | ------- |
+| no_info                      | error   |
+| mistyped_title_or_version    | error   |
+
```

Comment:
On second thought, I don't think we need any of this added to the readme. These are baseline spec requirements that don't really need documentation in my opinion. I'll ask Mike about this but for now, you can take this out